bogota-abril-2019/css-magazine-cover/Luis-Ruiz/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Magazine Cover</title>
+  </head>
+
+  <body>
+    <div class="bg-image"></div>
+
+    <section class="content">
+      <aside class="header">
+        <h4 class="header-left">December 2018</h4>
+        <h4 class="header-right">www.vibe.com</h4>
+      </aside>
+      <header>
+        <h1 class="title">vibe<span>.ng</span></h1>
+        <div class="title-secondary">
+          <h5>
+            THE RISE & RISE OF
+          </h5>
+          <h2>
+            YCEE
+          </h2>
+        </div>
+      </header>
+      <section class="info">
+        <aside class="info-left">
+          <span>6</span> REASONS WHY GIRLS ARE CRAZY ABOUT RUNTOWN'S MAD OVER
+          YOU
+        </aside>
+        <aside class="info-right">
+          <article class="info-right-anounce">
+            <h4 class="description">+</h4>
+          </article>
+          <article class="info-right-anounce">
+            <h4 class="info-right-anounce-title">
+              3 NIGERIAN FASHION DESIGNERS
+            </h4>
+            <h4 class="description">TO LOOKOUT FOR IN 2017</h4>
+          </article>
+          <article class="info-right-anounce">
+            <h4 class="info-right-anounce-title">TOP 10 NIGERIA</h4>
+            <h4 class="description">MUSIC VIDEOS OF 2016</h4>
+          </article>
+          <article class="info-right-anounce">
+            <h4 class="info-right-anounce-title">NEHRI OKWA</h4>
+            <h4 class="description">YOUNG GEO's MAKING IT</h4>
+          </article>
+        </aside>
+      </section>
+      <footer class="footer">
+        APPLE MUSIC LOCAL PRICING: A GOLDMINE FOR THE NEGERIAN MUSIC INDUSTRY

```

Comment:
Unnecessary used of Capital letter. Please use Text-transform CSS property.

https://css-tricks.com/almanac/properties/t/text-transform/

Reply:
Code Updated